### ISSUE TITLE
Why does pester need "Accessiblity" permissions?

### ISSUE THREAD

**Issue by pmetzger** [nriley/Pester#26]
Running Pester under MacOS 13.5.1, I get prompted to give Pester accessibility permissions. Any idea why?

**Comment by nriley** [comment]
Hi Perry — this is because Pester pops up its alert window on top of what you're doing, and it is trying to be considerate to make sure you're not typing when it pops anything up and end up typing your text into the Pester alert instead.  It'll delay the alert window, though not other alert options, for a bit.

You can see the code that does this [here](https://github.com/nriley/Pester/blob/master/Source/NSObject-PerformWhenIdle.m#L48) — simply asking macOS for the elapsed time since you last typed anything or moved the mouse.

If/when I get around to releasing another version of Pester (sorry, life has been very busy) I think I can add an explanation to this prompt. If you don't want/care about this feature you can feel free to deny access.

**Comment by pmetzger** [comment]
I think you should do a new release; it's been a while and Pester is remarkably useful.

**Comment by nriley** [comment]
If only it were that easy to find the time — my life is so different these days from 21 years ago when I started working on Pester! Thanks for sticking with it after all these years (I still use Pester daily myself).

Certainly going to try to give it a shot later this year.

**Comment by pmetzger** [comment]
Does doing a release require more than a few minutes? Would it help if I was to help you set up a github CI etc?

**Comment by upekkha** [comment]
Yes, please release a new version. The app has been [flagged as deprecated](https://github.com/Homebrew/homebrew-cask/commit/ec3b14d588ae3ffb9c3231c8ea4f6aa972b86784) in homebrew, as it looks unmaintained to them.

**Comment by billgothacked** [comment]
Good point:)
